app/[locale]/layout.tsx

```diff
@@ -60,13 +61,17 @@ export default function RootLayout({
   params: { locale },
 }: {
   children: React.ReactNode;
-  params: { locale: string };
+  params: { locale: Locale };
 }) {
   unstable_setRequestLocale(locale);
   const messages = useMessages();
 
+  let dir = 'ltr';
+  if (locale === 'he') {
+    dir = 'rtl';
+  }
```

Comment:
You could use the already exist hook for this, the hook is called [useTextDirection.ts](https://github.com/Maakaf/maakaf-website/blob/main/hooks/useTextDirection.ts) and can be found inside the hooks folder

Reply:
Nice! Will use